plugins/main/public/components/tools/devtools/application/components/dev-tools-action-buttons.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { EuiFlexGroup, EuiIcon } from '@elastic/eui';
+import { DEV_TOOLS_BUTTONS } from '../../constants';
+
+interface DevToolsActionButtonsProps {
+  onSendRequestButton: () => void;
+}
+
+const DevToolsActionButtons = ({
+  onSendRequestButton,
+}: DevToolsActionButtonsProps) => {
+  return (
+    <EuiFlexGroup
+      id={DEV_TOOLS_BUTTONS.ID}
+      justifyContent='flexEnd'
+      gutterSize='none'
+    >
+      <EuiIcon
+        type='play'
+        onClick={() => onSendRequestButton()}
+        title='Send request'
+        id={DEV_TOOLS_BUTTONS.PLAY_BUTTON_ID}
+        color='success'
+      />
+      <a href='' target='__blank' id={DEV_TOOLS_BUTTONS.DOCS_BUTTON_ID}>
```

Comment:
Add the `rel='noopener noreferrer'`. I think this is not present in the previous implementation but the links to documentation or external use it.

```suggestion
      <a href='' target='__blank' id={DEV_TOOLS_BUTTONS.DOCS_BUTTON_ID} rel='noopener noreferrer'>
```